File: CSharp Basics/01-First_Steps_In_Coding-lab/07-ProjectsCreation.cs
using System;

namespace ProjectsCreation
{
    class Program
    {
        static void Main(string[] args)
        {
            string architectName = Console.ReadLine();
            int projectsCount = int.Parse(Console.ReadLine());

            int hoursNeeded = projectsCount * 3;

            Console.WriteLine($"The architect {architectName} will need {hoursNeeded} hours to complete {projectsCount} project/s.");
        }
    }
}

File: C++ DEVELOPMENT/C++ Basics/01.First_Steps_In_Coding_Lab/07-Projects-Creation.cpp
#include <iostream>
#include <string>
using namespace std;

int main() 
{

	//1. въвеждаме име на архитект и брой проекти
	//2. изчисляваме време за изпълнение
	//3. отпечатваме: "The architect {името на архитекта} will need {необходими часове} hours to complete {брой на проектите} project/s."

	string architectName;
	cin >> architectName;

	int countProjects;
	cin >> countProjects;

	int time = countProjects * 3;

	cout << "The architect " << architectName << " will need " << time << " hours to complete " << countProjects << " project/s.";
}